### PR TITLE
style: add comments and small fixes

### PR DESCRIPTION
**Pull request by fedorov**

### CODE REVIEW

File: idc_index/index.py [hunk 1]
```diff
@@ -469,14 +469,15 @@ def get_viewer_URL(
         return viewer_url
 
     def _validate_update_manifest_and_get_download_size(
-        self, manifestFile, downloadDir, validate_manifest
+        self, manifestFile, downloadDir, validate_manifest=False
```
Comment (vkt1414) [inline idc_index/index.py:472]: this is an internal function that's called by download_from_manifest. So the validate_manifest is controlled at download_from_manifest. So this change must be reverted.

File: idc_index/index.py [hunk 2]
```diff
@@ -883,7 +891,7 @@ def download_from_manifest(
         manifestFile: str,
         downloadDir: str,
         quiet: bool = True,
-        validate_manifest: bool = True,
+        validate_manifest: bool = False,
```
Comment (vkt1414) [inline idc_index/index.py:894]: I'll do some benchmarking and update this. My hypothesis is there will be no difference, as  validate_manifest: bool = False does nothing but suppress much-needed warning messages when something is unusual in a manifest. As I have said here..s5cmd does not stop even with an invalid URL.

https://github.com/ImagingDataCommons/idc-index/pull/56#discussion_r1584075999

**Review by vkt1414 — CHANGES_REQUESTED** [review]

File: idc_index/index.py (hunk 2 above)
Comment (vkt1414) [reply idc_index/index.py:894]: here's the benchmark for validating the entire NLST collection and entire IDC data. As you can see, there is no difference between validation=True and validation =False. Please let me know if you still would like to turn it off by default. 

I have added a few print statements to let the user know, we are parsing the manifest.

**NLST**:
image

**IDC**:
image

Comment (fedorov) [reply idc_index/index.py:894]: This looks great! (obviously!)

**Comment by fedorov** [comment]
I can add those other smaller things later!

**Pull request closed without merging**